blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/BluefloodEnumRollup.java

```diff
@@ -46,4 +58,24 @@ public boolean equals(Object obj) {
         return hashedEnumValues2Count.equals(other.hashedEnumValues2Count);
     }
 
+    public static BluefloodEnumRollup buildRollupFromEnumRollups(Points<BluefloodEnumRollup> input) throws IOException {
+        BluefloodEnumRollup enumRollup = new BluefloodEnumRollup();
+        for (Points.Point<BluefloodEnumRollup> point : input.getPoints().values()) {
+            BluefloodEnumRollup pointData = point.getData();
+            Map<Long, Long> incomingHashedEnums = pointData.getHashedEnumValuesWithCounts();
+            Map<Long, Long> currentHashedEnums = enumRollup.getHashedEnumValuesWithCounts();
```

Comment:
It seems redundant to put this line in the for loop.  Wouldn't it work the same if currentHashedEnums was set once outside the loop? or am i missing something